spec/facility_factory_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'spec_helper.rb'
+
+RSpec.describe FacilityFactory do
+
+    before(:each) do
+        @factory = FacilityFactory.new
+
+        @colorado_facilities = DmvDataService.new.co_dmv_office_locations
+        @new_york_facilities = DmvDataService.new.ny_dmv_office_locations
+        @missouri_facilities = DmvDataService.new.mo_dmv_office_locations
+    end
+
+    describe '# initialize' do
+
+        it 'can initialize' do
+            expect(@factory).to be_an_instance_of FacilityFactory
+        end
+
```

Comment:
Too many white spaces. No white spaces between ends and don't need one between describe and next it block.